core/src/main/java/com/netflix/iceberg/util/BinPacking.java

```diff
@@ -78,13 +84,15 @@ public PackingIterable(Iterable<T> iterable, long targetWeight, int lookback,
     private final long targetWeight;
     private final int lookback;
     private final Function<T, Long> weightFunc;
+    private final boolean largestBinFirst;
 
     private PackingIterator(Iterator<T> items, long targetWeight, int lookback,
-                            Function<T, Long> weightFunc) {
+                            Function<T, Long> weightFunc, boolean largestBinFirst) {
```

Comment:
Minor: this doesn't seem to follow standard indentation, the line should wrap to 2 spaces forward from previous indentation.

Reply:
This follows the conventions I was using for a while. I think it is fine in this PR and we will fix the style when we standardize on one format.

Reply:
Sure. I assumed we had standardized it. Will bear in mind.